src/test/java/org/takes/misc/VerboseIterableTest.java

```diff
@@ -0,0 +1,47 @@
+package org.takes.misc;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import static org.junit.Assert.*;
+
+/**
+ * Test case for {@link VerboseIterable}
+ * @author Zarko Celebic
+ */
+public class VerboseIterableTest {
+
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
+
+    @Test
+    public void testIterator() throws Exception {
```

Comment:
@celezar see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names for test name and test description.